content/influxdb/v2.0/upgrade/v1-to-v2/docker.md

```diff
@@ -0,0 +1,245 @@
+---
+title: Upgrade from InfluxDB 1.x to 2.0 with Docker
+description: >
+  Use the automated upgrade process built into the InfluxDB 2.x Docker image to
+  update InfluxDB 1.x Docker deployments to InfluxDB 2.x.
+menu:
+  influxdb_2_0:
+    parent: InfluxDB 1.x to 2.0
+    name: Upgrade with Docker
+weight: 101
+---
+
+Use the automated upgrade process built into the [InfluxDB 2.x Docker image](https://hub.docker.com/_/influxdb)
+to update InfluxDB 1.x Docker deployments to InfluxDB 2.x.
+
+- [Upgrade requirements](#upgrade-requirements)
+- [Upgrade examples](#upgrade-examples)
+
+{{% note %}}
+#### Export continuous queries before upgrading
+The automated upgrade process **does not** migrate InfluxDB 1.x continuous queries (CQs)
+to InfluxDB 2.x tasks. Export all of your CQs before upgrading to InfluxDB 2.x.
+For information about exporting and migrating CQs to tasks, see
+[Migrate continuous queries to tasks](/influxdb/v2.0/upgrade/v1-to-v2/migrate-cqs/).
+{{% /note %}}
+
+## Upgrade requirements
+InfluxDB 2.x provides a 1.x-compatible API, but expects a different storage layout on disk.
+To account for theses differences, the InfluxDB Docker image contains extra functionality to migrate
+1.x data and configuration into 2.x layouts automatically before booting the `influxd` server.
```

Comment:
The terms "configuration" and "layouts" are confusing here. Perhaps:
```suggestion
1.x data and into 2.x-compatible data automatically before booting the `influxd` server.
```

Reply:
Maybe this?
```suggestion
1.x files into a 2.x-compatible structure automatically before booting the `influxd` server.
```

"Configuration" here is the config file.